src/components/safe-messages/MsgModal/index.tsx

```diff
@@ -111,6 +112,8 @@ const MsgModal = ({
         await dispatchSafeMsgConfirmation({ signer, safe, message: decodedMessage, requestId })
       }
 
+      trackEvent(MESSAGE_MODAL_EVENTS.SIGN_MESSAGE)
```

Comment:
Please don't trigger events like this, there's `useTxTracking` that subscribes to events and tracks.